src/items/deleteItemDialog.component.test.tsx

```diff
@@ -30,12 +30,14 @@ describe('delete item dialog', () => {
   };
 
   beforeEach(() => {
-    item = getItemById('KvT2Ox7n');
+    item = { ...getItemById('KvT2Ox7n') }; // prevents item state leaking between tests when mutating it's properties
```

Comment:
```suggestion
    item = { ...getItemById('KvT2Ox7n') };
```
Good spot, Create an issue for this as if this is a problem here is should be change everywhere,